src/transactions/feed/TransactionFeed.tsx

```diff
@@ -30,8 +29,7 @@ function TransactionFeed() {
   const { loading, error, transactions, fetchingMoreTransactions, fetchMoreTransactions } =
     useFetchTransactions()
 
-  const cachedTransactions = useSelector(transactionsSelector)
```

Comment:
nothing interesting is changed in this file. the cachedTransactions is removed here because it will come with the `transactions` from the `useFetchTransactions` hook